spec/actions/update_attribute_actions_spec.rb

```diff
@@ -0,0 +1,83 @@
+describe Magma::UpdateAttributeAction do
+  let(:project_name) { 'labors' }
```

Comment:
Does this spec need to replace attributes like the ones in update_model_spec.rb?

Reply:
Can you clarify what you mean here?

Reply:
In the update_model_spec.rb you are careful to preserve the original state of the attribute before you modify it.

    original_attribute = Labors::Monster.attributes[:name].dup
    ...
    Labors::Monster.attributes[:name] = original_attribute

 Here you modify the (same) original attribute, but are not careful about preserving the original state?

Reply:
This test doesn't make any changes to the attribute because the action is mocked out. When we call `#perform` on the action, instead of making changes to the attribute it just returns true.

```ruby
let(:update_attribute_action) { double('update_attribute') }
let(:perform_return) { true }
allow(update_attribute_action).to receive(:perform).and_return(perform_return)
```